{'project_id': '3130', 'year': '1984', 'x': -563730, 'y': 2537520, 'plot_id': '8251', 'strata': '41'}
Pandas(Index=0, project_id='3130', plot_id='8251', x=-563730, y=2537520)
called process_plot
Pandas(Index=0, project_id='3130', plot_id='8251', x=-563730, y=2537520)
8251
Directory '/efs/timesync/1984/audit/' already exists.
Data has been pickled and saved to '/efs/timesync/1984/audit/1984_8251_3130.p'.
..............Failed qa: 1984-04-12T16:57:05.6160190Z
Failed qa: 1984-05-05T17:03:23.4270130Z
Failed qa: 1984-05-12T17:09:48.4160560Z
Failed qa: 1984-05-14T16:57:06.7080250Z
Failed qa: 1984-05-28T17:10:15.1690810Z
............................Failed qa: 1984-07-01T16:58:14.213556Z
..............Failed qa: 1984-07-17T16:58:30.767075Z
.....................Failed qa: 1984-09-03T16:59:47.663553Z
.......Failed qa: 1984-09-19T16:59:38.3860250Z
Failed qa: 1984-09-26T17:06:13.2430500Z
.....................Failed qa: 1984-11-22T16:59:36.368535Z
Data has been pickled and saved to '/efs/timesync/1984/audit/1984_8251_3130_qa_rejects.p'.
Success tracking goes here
